charts/hail-search/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for hail_search.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 4
+
+image:
+  repository: gcr.io/seqr-project/seqr-hail-search
+  pullPolicy: Always
+  # -- The docker image tag to pull from the repository
+  # tag: "gcloud-dev"
+
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+service:
+  type: ClusterIP
+  port: 5000
+
+resources: 
+  {}
+
+# default environment variables
+environment:
+  DATASETS_DIR: '/datasets'
+
+sourceDatasetsDir: ''
+datasetVersions:
+  GRCh37/SNV_INDEL: ''
+  GRCh38/SNV_INDEL: ''
```

Comment:
for the open source chart, maybe lets only provide the SNV_INDEL paths by default